### PR TITLE
Add Echelon (ECH) and IBC Channels

### PR DESCRIPTION
**Pull request by echelonfoundation**
IBC relayers are up and running (three or so on the ones committed) and transactions work

Official Site: https://ech.network
Official Dapp: https://app.ech.network

### CODE REVIEW

**Comment by echelonfoundation** [comment]
We wanted to add that we have completed every requirement on your list in documentation for Osmosis listing.

We have created an OSMO/ECH LP that contains over $1,381 in liquidity currently. Pool #747: https://app.osmosis.zone/pool/747

0.2% Swap Fee with 50/50 OSMO/ECH

You can deposit/withdraw to Osmosis via our dapp at https://app.ech.network/ibc (Echelon only works within Metamask currently) or can be done via CLI.

Thank you!

File: packages/web/config/ibc-assets.ts [hunk 1]
```diff
@@ -717,6 +717,15 @@ export const IBCAssetInfos: (IBCAsset & {
...
     coinMinimalDenom: "ukuji",
   },
+  {
+    counterpartyChainId: "echelon_3000-3",
+    sourceChannelId: "channel-262",
+    destChannelId: "channel-8",
+    coinMinimalDenom: "aechelon",
+    depositUrlOverride: "https://app.ech.network/ibc",
+    withdrawUrlOverride: "https://app.ech.network/ibc",
+    isVerified: true,
```
Comment (sunnya97) [inline packages/web/config/ibc-assets.ts:727]:
```suggestion
```

Comment (sunnya97) [reply packages/web/config/ibc-assets.ts:727]: We will not be adding this asset to the main frontend rn.  Only the permissionless frontier.

Comment (echelonfoundation) [reply packages/web/config/ibc-assets.ts]: Ack and updated! Thank you @sunnya97

Comment (jonator) [reply packages/web/config/ibc-assets.ts]: Conventionally, we only include assets on the main frontend that are receiving incentives. 

Though, with current market conditions we are proposing to revisit our incentive strategy by redirecting some to the community pool. @sunnya97 thoughts?

File: packages/web/config/chain-infos.ts [hunk 2]
```diff
@@ -1338,6 +1338,34 @@ const chainInfos = (
...
+        coinType: 60,
+      },
+      bech32Config: Bech32Address.defaultBech32Config("echelon"),
+      currencies: [
+        {
+          coinDenom: "ECH",
+          coinMinimalDenom: "aechelon",
+          coinDecimals: 18,
+          coinGeckoId: "echelon",
+          coinImageUrl: "/tokens/ech.png",
```
Comment (JeremyParish69) [inline packages/web/config/chain-infos.ts:1356]: the image file must be square aspect ratio, please

File: packages/web/config/chain-infos.ts [hunk 3]
```diff
@@ -1338,6 +1338,34 @@ const chainInfos = (
       features: ["stargate", "ibc-transfer", "no-legacy-stdTx", "ibc-go"],
       explorerUrlToTx: "https://www.mintscan.io/evmos/txs/{txHash}",
     },
+    {
```
Comment (JeremyParish69) [inline packages/web/config/chain-infos.ts:1341]: can we move this to the bottom of the list please? Not sure why it's placed here.

**Pull request merged**